src/dsp_tools/xmllib/helpers.py

```diff
@@ -1143,8 +1141,21 @@ def find_license_in_string(string: str) -> License | None:
         - "inconnu" -> LicenseRecommended.DSP.UNKNOWN
         - "CC BY" -> LicenseRecommended.CC.BY
         - "Creative Commons BY 4.0" -> LicenseRecommended.CC.BY
+        - "CC 0 1.0" -> LicenseOther.Public.CC_0_1_0
+        - "CC PDM 1.0" -> LicenseOther.Public.CC_PDM_1_0
+        - "BORIS Standard License" -> LicenseOther.Various.BORIS_STANDARD
+        - "LICENCE OUVERTE 2.0" -> LicenseOther.Various.FRANCE_OUVERTE
     """
-    sep = r"[- _]+"
+    if lic := _get_already_parsed_license(string):
+        return lic
+
+    sep = r"[-_\p{Zs}]+"  # Zs = unicode category for space separator characters
```

Comment:
adding `\p{Zs}` gives us more flexibility regarding the separator character. For example, it allows non-breaking spaces and other whitespaces.